tests/common.go

```diff
@@ -617,49 +619,51 @@ func getStorageNodes() ([]node.Node, error) {
 // CollectSupport creates a support bundle
 func CollectSupport() {
 	context(fmt.Sprintf("generating support bundle..."), func() {
-		Step(fmt.Sprintf("save all useful logs on each node"), func() {
-			nodes := node.GetWorkerNodes()
-			expect(nodes).NotTo(beEmpty())
+		nodes := node.GetWorkerNodes()
+		expect(nodes).NotTo(beEmpty())
 
-			for _, n := range nodes {
-				if !n.IsStorageDriverInstalled {
-					continue
-				}
+		for _, n := range nodes {
+			if !n.IsStorageDriverInstalled {
+				continue
+			}
+			Step(fmt.Sprintf("save all useful logs on node %s", n.SchedulerNodeName), func() {
 
-				logrus.Infof("collecting diags from %s", n.Name)
 				Inst().V.CollectDiags(n)
 
 				journalCmd := fmt.Sprintf("journalctl -l > %s/all_journal_%v.log", Inst().BundleLocation, time.Now().Format(time.RFC3339))
-				logrus.Infof("saving journal output on %s", n.Name)
 				runCmd(journalCmd, n)
 
-				logrus.Infof("saving portworx journal output on %s", n.Name)
 				runCmd(fmt.Sprintf("journalctl -lu portworx* > %s/portworx.log", Inst().BundleLocation), n)
 
-				logrus.Infof("saving kubelet journal output on %s", n.Name)
 				Inst().S.SaveSchedulerLogsToFile(n, Inst().BundleLocation)
 
-				logrus.Infof("saving dmesg output on %s", n.Name)
 				runCmd(fmt.Sprintf("dmesg -T > %s/dmesg.log", Inst().BundleLocation), n)
 
-				logrus.Infof("saving disk list on %s", n.Name)
 				runCmd(fmt.Sprintf("lsblk > %s/lsblk.log", Inst().BundleLocation), n)
 
-				logrus.Infof("saving mount list on %s", n.Name)
 				runCmd(fmt.Sprintf("cat /proc/mounts > %s/mounts.log", Inst().BundleLocation), n)
 
-				// this is a small tweak especially for providers like openshift, aws where oci-mon saves this file
-				// with root read permissions only but collect support bundle is a non-root user
-				runCmd(fmt.Sprintf("chmod 755 %s/oci.log", Inst().BundleLocation), n)
-			}
-		})
+				// if oci.log is present, this is a small tweak especially for providers like openshift, aws where
+				// oci-mon saves this file with root read permissions only but collect support bundle is a non-root user
+				statCmd := fmt.Sprintf("stat %s/oci.log", Inst().BundleLocation)
+
+				_, err := Inst().N.RunCommand(n, statCmd, node.ConnectionOpts{
+					Timeout:         defaultCmdTimeout,
+					TimeBeforeRetry: defaultCmdRetryInterval,
+					Sudo:            true,
+				})
+				if err == nil {
+					runCmd(fmt.Sprintf("chmod 755 %s/oci.log", Inst().BundleLocation), n)
+				}
+			})
+		}
 	})
 }
```

Comment:
looks like this PR has unrelated changes other than just improving pod logging. Let's separate them out.

Reply:
done